Documentation/getting-started.md

```diff
@@ -39,3 +39,21 @@ Sample data is available for download from the [SlicerAutoscoperM Sample Data](/
 * Ankle BVR data - This data was provided by Michael Rainbow.
   * Three frames of movement are included in the sample data.
   * Three partial volumes are included in the sample data. The tibia, talus, and calcaneus are included.
+
+## Glossary
+
+This section defines terms that are commonly used by Autoscoper and SlicerAutoscoperM
+
+* **3D Computed Tomography (CT)**: Computerized x-ray imaging method producing volumetric density data of a region of interest. Saved in DICOM format.
+* **4D CT**: Temporal 3D CT
+* **Biplaner Videoradiography (BVR)**: Two calibrated x-ray sources and high-speed video camera sets that allow for the synchronous collection of radiographs from two perspectives. Camera models from the calibrations allow for unified 3D World construction from perspective 2D views.
+* **Rigid Body**: Commonly a bone or an implant. The rigid body is the region of interest within a 3DCT. Users segment or isolate the data which represents the rigid body. Often the rigid body surface file is exported as a bone mesh model (STL). Autoscoper is used to track the kinematics (rotation and translation) of the rigid body in 3-dimensional space.
+* **Partial Volume (PVOL)**: Saved as a TIFF stack, the PVOL is a volumetric image data file. A PVOL is a subset of 3DCT scan data as defined from a segmented rigid body and cropped to its dimensions.
+* **AUT Model (AUT)**: A representation of the rigid body STL as parsed in the Autoscoper 3D World
+* **Digitally Reconstructed Radiograph (DRR)**: When a version of the PVOL is loaded into Autoscoper, the perspective camera models cast the volumetric density data onto the same image plane as the radiographs to generate a synthetic radiograph
+* **Radiograph**: A 2D image produced by x-rays.
+* **Sequential 3D CT**: A collection of 3D CT scans acquired of the same region of interest (joint or limb) in different positions. Using the Sequences model, the independent volumes can be visualized and processed together in a series.
```

Comment:
```suggestion
* **Sequential 3D CT**: A series of 3D CT scans acquired from the same region of interest (such as a joint or limb) in different positions. Using the Sequences module, these independent volumes can be visualized and processed together as a series.
```

I am assuming `Sequences module` should be used instead of `Sequences model`